.github/workflows/tests-nightly.yml

```diff
@@ -116,7 +119,7 @@ jobs:
           node-version: 14
 
       - name: Cypress run
-        if: ${{ github.event.inputs.tests != 'Critical' }}
+        if: ${{ github.event.inputs.tests != 'Critical' && github.event_name=! 'repository_dispatch'}}
```

Comment:
In multiple place I see `=!` instead of `!=`

Reply:
Fixed, I don't know how I did that